gensim/models/tfidfmodel.py

```diff
@@ -127,11 +162,6 @@ def initialize(self, corpus):
 
         # and finally compute the idf weights
         n_features = max(dfs) if dfs else 0
-        logger.info(
```

Comment:
why you remove this?

Reply:
This showed the progress of the `pre - computation` step. Now that the values are no longer being pre-computed I think we need to log this in a different way. How do you want it to go?